### DISCUSSION TITLE
Support policy not updated with v5 release

### DISCUSSION THREAD

**Discussion by SayakMukhopadhyay** [vmware/clarity#5792] [Ideas] [answered]
The support policy at **https://clarity.design/get-started/support/#support-policy** still shows that v5 is in pre-release. Can someone update that so that current state is accurately reflected.

**Comment by gnomeontherun** [comment, marked as answer]
Yes I am going to do this now, expect it to be updated soon.

**Reply by bbogdanov** [reply]
https://github.com/vmware/clarity/pull/5801 The release date was added with this PR
